bot/build.gradle

```diff
@@ -44,7 +45,11 @@ dependencies {
     implementation "com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:$jacksonVersion"
 
-    implementation "ch.qos.logback:logback-classic:$logbackVersion"
+    implementation (group: "ch.qos.logback", name: "logback-classic") {
```

Comment:
Are named parameters required here? Can't you just put `implementation ("ch.qos.logback", "logback-classic")` ?

Also, why is the strict version required here?

Reply:
Named parameter are needed or else it calls a different function. Strict version is required because else both the fork and the original one was added, idk why